config/default/manager_image_patch.yaml

```diff
@@ -8,6 +8,7 @@ spec:
     spec:
       containers:
       # Change the value of image field below to your controller image URL
+      - image: gcr.io/vincepri-dev/cluster-api-gcp-controller-amd64:dev
```

Comment:
```suggestion
```

Reply:
This shouldn't be removed entirely, we should use the staging bucket as it was before

Reply:
The image is listed twice